Packs/Akamai_WAF/ReleaseNotes/2_2_0.md

```diff
@@ -0,0 +1,19 @@
+
+#### Integrations
+
+##### Akamai WAF
+
+- Added support for **akamai-list-datastream-groups** command that returns access groups with contracts on your account. you can later use the groupid and contractid values to create and view streams or list properties by group. set the contractid query parameter to get groups for a specific contract.
```

Comment:
You can later use the groupid and contractid values to create and view streams or list properties by group. Set the contractid query parameter to get groups for a specific contract.